test/boost/wasm_alloc_test.cc

```diff
@@ -0,0 +1,94 @@
+#include "lang/wasm.hh"
+#include "lang/wasm_instance_cache.hh"
+#include "rust/wasmtime_bindings.hh"
+#include "seastar/core/reactor.hh"
+#include <seastar/testing/test_case.hh>
+#include <seastar/core/coroutine.hh>
+
```

Comment:
Missing copyright/license. Please check other new files too.

Better to consolidate wasm tests in fewer files, to reduce bloat on disk as our executables are large.

Reply:
I added the license to the files where it was missing. I did try to put all wasm-related tests in one file, but for the reasons described in this test file, it was not possible when testing traps in wasm. If we make more tests for wasm traps, we may need to prepare a workaround for this to avoid creating a new test file for each of such tests